hooks/after_platform_add/ios/ios_move_locales.js

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env node
+
+var fs = require('fs-extra');
+var path = require('path');
+
+const LOG_NAME = "Adding locales: ";
+
+module.exports = function (context) {
+    // If ios platform is not installed, don't even execute
+    var localesFolder = path.join(context.opts.projectRoot, 'locales/');
+
+    if (context.opts.cordova.platforms.indexOf('ios') < 0 || !fs.existsSync(localesFolder))
+        return;
+    
+    console.log(LOG_NAME + "Retrieving application name...")
+    var config_xml = path.join(context.opts.projectRoot, 'config.xml');
+    var et = context.requireCordovaModule('elementtree');
+    var data = fs.readFileSync(config_xml).toString();
+    // If no data then no config.xml
+    if (data) {
+        var etree = et.parse(data);
+        var applicationName = etree.findtext('./name');
+        console.log(LOG_NAME + "Your application is " + applicationName);
+        var localesFolder = path.join(context.opts.projectRoot, 'locales/');
+
+        var languagesFolders = fs.readdirSync(localesFolder);
+        // It's not problematic but we will remove them to have cleaner logs.
+        var filterItems = ['.git', 'LICENSE', 'README.md']
+        languagesFolders = languagesFolders.filter(item => !filterItems.includes(item));
+        console.log(LOG_NAME + "Languages found -> " + languagesFolders);
+        languagesFolders.forEach(function (language) {
+            console.log(LOG_NAME + 'I found ' + language + ", I will now move the files.")
+            var platformRes = path.join(context.opts.projectRoot, 'platforms/ios/' + applicationName + "/Resources/");
+            var wwwi18n = path.join(context.opts.projectRoot, 'www/i18n/');
+            var languageFolder = localesFolder + "/" + language;
+
+            var lproj = "/" + language + ".lproj";
+            var lprojFolder = path.join(languageFolder, lproj);
+            if (fs.existsSync(lprojFolder)) {
+                console.log(LOG_NAME + "Moving " + lprojFolder + " to " + platformRes);
+
+                var platformlproj = platformRes + lproj;
+                if (!fs.existsSync(platformlproj)) {
+                    console.log(LOG_NAME + platformlproj + "does not exist, I will create it.");
+                    fs.mkdirSync(platformlproj);
+                }
+
+                fs.copySync(lprojFolder, platformlproj);
```

Comment:
So I am not quite sure what you mean by "I didn't found a way to dynamically add "Localization" to iOS ". What is not working? 

Are you worried about the "Base Internationalization"? It looks like that is mainly focused around automatically extracting user-facing strings from 

> Base internationalization separates user-facing strings from .storyboard and .xib files. It relieves localizers of the need to modify .storyboard and .xib files in Interface Builder....

We don't really have complex storyboards or xib files - the storyboard just uses a webview and the rest of the UX is HTML + CSS + JS.

The only user visible strings are the ones that you pulled out into the strings files. So you really need the "Creating Strings Files for User-Facing Text in Your Code" section of [maintaining your own strings files](https://developer.apple.com/library/archive/documentation/MacOSX/Conceptual/BPInternational/MaintaingYourOwnStringsFiles/MaintaingYourOwnStringsFiles.html#//apple_ref/doc/uid/10000171i-CH19-SW22)

and that basically just outlines this procedure, with the caveat that you can use `genstrings` if needed to autogenerate the strings file by looking at `NSLocalizedString`. But you have already manually generated this so I think you are good.

If you have more details on what exactly you are worried about, I can see if I can suggest something.